plugins/inputs/nats_consumer/README.md

```diff
@@ -17,6 +17,9 @@ instances of telegraf can read from a NATS cluster in parallel.
   ## subject(s) to consume
   subjects = ["telegraf"]
 
+  ## jetstream subjects
+  js_subjects = ["js_telegraf"]
```

Comment:
Can you please make this `jetstream_subjects` to not make the user search for abbreviations!?